content/docs/k8s/quickstart.mdx

```diff
@@ -0,0 +1,280 @@
+  namespace/pomerium configured
+  customresourcedefinition.apiextensions.k8s.io/pomerium.ingress.pomerium.io created
+  serviceaccount/pomerium-controller created
+  serviceaccount/pomerium-gen-secrets created
+  clusterrole.rbac.authorization.k8s.io/pomerium-controller created
+  clusterrole.rbac.authorization.k8s.io/pomerium-gen-secrets created
+  clusterrolebinding.rbac.authorization.k8s.io/pomerium-controller created
+  clusterrolebinding.rbac.authorization.k8s.io/pomerium-gen-secrets created
+  service/pomerium-metrics created
+  service/pomerium-proxy created
+  deployment.apps/pomerium created
+  job.batch/pomerium-gen-secrets created
+  ingressclass.networking.k8s.io/pomerium created
+  ```
+
+1. Define a Kubernetes secret to hold identity provider (**IdP**) credentials:
+
+  ```yaml title=idp-secret.yaml
+  apiVersion: v1
+  kind: Secret
+  metadata:
+    name: idp
+    namespace: pomerium
+  type: Opaque
+  stringData:
+    client_id: ${IDP_PROVIDED_CLIENT_ID}
+    client_secret: ${IDP_PROVIDED_CLIENT_SECRET}
+  ```
+
+  The data required in this secret will depend on your [identity provider]. See the [Kubernetes Reference](/docs/k8s/reference#identityprovider) page for all available keys, and cross-reference it with the requirements of your IdP.
+
+1. Provide this secret to the cluster:
+
+  ```sh
+  kubectl apply -f idp-secret.yaml
+  ```
+
+  You can confirm with `kubectl get secrets --namespace pomerium`:
+
+  ```sh title=settings.yaml {4}
+  NAME                               TYPE                                  DATA   AGE
+  bootstrap                          Opaque                                3      99m
+  default-token-whpnw                kubernetes.io/service-account-token   3      149m
+  idp                                Opaque                                2      5s
+  pomerium-controller-token-cckv5    kubernetes.io/service-account-token   3      99m
+  pomerium-gen-secrets-token-4pdkd   kubernetes.io/service-account-token   3      99m
+  ```
+
+1. Define the global Pomerium settings:
+
+  By default, Pomerium expects the configuration document `global`, which may be customized via command line arguments to the container, if necessary. 
+
+  ```yaml title=pomerium.yaml
+  apiVersion: ingress.pomerium.io/v1
+  kind: Pomerium
+  metadata:
+    name: global
+  spec:
+    secrets: pomerium/bootstrap
+    authenticate:
+      url: https://authenticate.localhost.pomerium.io
+    identityProvider:
+      provider: ${YOUR_IdP}
+      secret: pomerium/idp
+    certificates:
+      - pomerium/pomerium-wildcard-tls
+  ```
+
+1. Apply the global settings:
+
+  ```sh
+  kubectl apply -f pomerium.yaml
+  ```
+
+  The Pomerium Proxy service should now be running in your cluster:
+
+  ```sh {3}
+  kubectl get service pomerium-proxy
+  NAME             TYPE           CLUSTER-IP   EXTERNAL-IP     PORT(S)                      AGE
+  pomerium-proxy   LoadBalancer   10.24.7.1    198.51.100.228   443:31498/TCP,80:31437/TCP   10s
+  ```
```

Comment:
Pomerium-proxy service is created from the very beginning, it just won't have endpoints until then. 
We should advise users to do `kubectl describe pomerium/global` to get Pomerium status.